ambry-store/src/main/java/com.github.ambry.store/StoreDescriptor.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.utils.Utils;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.UUID;
+
+
+/**
+ * Refers to the Store descriptor of a {@link BlobStore} having some metadata information about each store.
+ * As of now, store descriptor has storeId and incarnation Id. IncarnationId is a unique identifier for every new
+ * incarnation of the store. The contents of store descriptor is expected to evolve.
+ */
+class StoreDescriptor {
+  String storeId;
+  UUID incarnationId;
+  public static final int VERSION = 0;
```

Comment:
also this has to be `short` (the very next variable says that `VERSION_SIZE` is 2).